lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -190,6 +194,7 @@ if (typeof module !== 'undefined' && module.exports) {
 /**
  * @typedef {{
  *     type: string,
+ *     styleAs: (string|undefined),
```

Comment:
I didn't see it as table-only we just didn't have a need for text styling yet, should I add back in the class to the text element too?